SU2_CFD/include/limiters/CLimiterDetails.hpp

```diff
@@ -155,7 +155,7 @@ struct CLimiterDetails<VENKATAKRISHNAN>
 template<>
 struct CLimiterDetails<VENKATAKRISHNAN_WANG>
 {
-  su2activematrix fieldMin, fieldMax;
+  static su2activevector sharedMin, sharedMax;
   su2activevector eps2;
```

Comment:
For this limiter we need a min/max of the field over the entire domain, computing these reductions with OpenMP requires some shared variables that all threads can access. Since all threads call `computeLimiters`, the `CLimiterDetails` object(s) will be thread-private, the cleanest way I could think of to have shared state was to make the variables static.